routers/routes/routes.go

```diff
@@ -486,6 +486,21 @@ func RegisterRoutes(m *macaron.Macaron) {
 				m.Post("/title", repo.UpdateIssueTitle)
 				m.Post("/content", repo.UpdateIssueContent)
 				m.Post("/watch", repo.IssueWatch)
+				m.Post("/addDependency", repo.AddDependency)
+				m.Post("/removeDependency", repo.RemoveDependency)
```

Comment:
It should be 1 endpoint (`/dependency`), 1 function and make decision based on parameter (eg `?action=add` and `?action=remove`).

Reply:
And why not use the http verbs (`post` and `delete`)? So, the action isn't needed

Reply:
Or better and consistent way: `/dependency/add` and `/dependency/delete`

Reply:
Top solution for me would be creating `dependency` api and just call that with simple ajax. Clear, no redirects, fast, low data consumption...

Reply:
I'm reading the routes.go and yeah, it seems the most consistent way with other routes :+1:  However, I think the best way to do it is using the http verbs :laughing:

Reply:
Doing it only via UI would require a lot of work to update the comment section afterwards, I think this is out of scope for this pr. Also i think there is already an issue for this (make gitea load contents more dynamically).

But I admit, would be very cool.

Reply:
Nevertheless, I optimized the routes.